### DISCUSSION TITLE
API余额用完了怎么办，哪里可以买吗

### DISCUSSION THREAD

**Discussion by sparkmy181** [GaiZhenbiao/ChuanhuChatGPT#432] [General]
API余额用完了怎么办，哪里可以买吗

**Comment by GaiZhenbiao** [comment]
可以用虚拟信用卡消费

**Reply by Git179822868** [reply]
> API余额用完了怎么办，哪里可以买吗
直接打开https://platform.openai.com/account/billing/payment-methods
如果嫌麻烦我可以给你一个api

**Reply by sparkmy181** [reply]
啊，用这个api不是要钱的吗

**Reply by Git179822868** [reply]
我目前有一个5美金的号给你，后续买我可以多注册几个

**Reply by Git179822868** [reply]
> 啊，用这个api不是要钱的吗

都说给你的还要什么钱

**Reply by sparkmy181** [reply]
感谢🙏兄弟，我怎么联系你

**Reply by Git179822868** [reply]
> 感谢🙏兄弟，我怎么联系你
加我微信吧T-ENE-T

**Comment by learnCodeON** [comment]
你可以学习一下“浏览器高级搜索”，比如：如果你想知道“chatgpt账号是怎么充值的”，你可以在浏览器里面搜索“intitle:chatgpt账号+充值”或者“intitle:chatgpt账号+购买”。
